entities/communication/quota.json

```diff
@@ -0,0 +1,37 @@
+{
+  "type": "object",
+  "properties": {
+    "business_uid": {
+      "type": "string",
+      "description": "The business that the quota relates to."
+    },
+    "quota_uid": {
+      "type": "string",
+      "description": "The entity's unique identifier."
+    },
+    "remaining_minutes": {
+      "type": "integer",
+      "description": "The remaining minutes for calls."
+    },
+    "source": {
+      "type": "string",
+      "description": "The quota creator."
```

Comment:
is this an enum? By the looks of the example, seems like there is a set of values.